editor/modes/visual-editor/block.js

```diff
@@ -148,6 +167,8 @@ class VisualEditorBlock extends wp.element.Component {
 				onMouseMove={ this.maybeHover }
 				onMouseLeave={ onMouseLeave }
 				className={ className }
+				data-type={ block.blockType }
+				{ ...getBlockAttributesAsProps( block.attributes ) }
```

Comment:
It's a good name for now but could be changed later if we settle on changing the `edit` name